src/main/java/frc/robot/commandGroups/TapeCommandGroup.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.commandGroups;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.autonomous.FollowPath;
+import frc.robot.subsystems.arm.ArmConstants;
+import frc.robot.subsystems.arm.commands.SetArmsPosition;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.subsystems.gyroscope.Gyroscope;
+import frc.robot.subsystems.vision.Limelight;
+
+public class TapeCommandGroup extends SequentialCommandGroup {
+    public TapeCommandGroup() {
+        addCommands(
+                FollowPath.generatePathToAprilTag(new SwerveDrive(), Limelight.getInstance(), new Gyroscope())
```

Comment:
You shouldn't instantiate subsystems each time you use this command.
This will also not work, because you can't instantiate hardware objects (e.g. motor controllers) more than once (this would also not make sense).
Also, the Limelight object should be passed as a parameter.

Reply:
The class was deleted